src/main/java/com/puppetlabs/jenkins/plugins/puppetgatling/gatling/PuppetGatlingBuildAction.java

```diff
@@ -67,7 +67,15 @@ public ReportRenderer getReport(String simulationName) {
 	}
 
 	public String getReportURL(String simulationName) {
-		return new StringBuilder().append(URL_NAME).append("/report/").append(simulationName).toString();
+		return new StringBuilder().append("job/").
```

Comment:
Works for me.  I tested this locally and all seems to work properly.  So, I'm 👍.  Merging...